src/NServiceBus.Metrics.AcceptanceTests/Tests/When_reporting_to_ServiceControl.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace NServiceBus.Metrics.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using global::Metrics.Json;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_reporting_to_ServiceControl : NServiceBusAcceptanceTest
+    {
+        static string MonitoringSpyAddress => Conventions.EndpointNamingConvention(typeof(MonitoringSpy));
+
+        [Test]
+        public async Task Should_send_reports_to_configured_queue()
```

Comment:
This test is just asserting that the message arrives. Do we need to make any sort of assertions about the data?